src/Lykke.Blockchains.Integrations.Contract/Common/Base58String.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Text;
+using System.Text.RegularExpressions;
+using JetBrains.Annotations;
+using Newtonsoft.Json;
+using SimpleBase;
```

Comment:
[Multiformat's multibase](https://github.com/multiformats/multibase) looks like more mature project.

Reply:
Yep, looks so. Switched to multibase.